scripts/Xvfb-bench-run.sh

```diff
@@ -47,12 +49,12 @@ function program_exit() {
 function bench_terminal() {
     printf "\033[1m==> Running terminal: $1\033[m\n"
     local terminal_name=$(basename $1)
-    time "${@}" -e "${TB_BIN}" --fixed-size --stdout-fastpath --column-by-column --output "${OUTPUT_DIR}/${terminal_name}_results"
+    time "${@}" -e "${TB_BIN}" --fixed-size --column-by-column --size 2 --output "${OUTPUT_DIR}/${terminal_name}_results"
     local exit_code=$?
     printf "\033[1m==> Terminal exit code: $exit_code\033[m\n"
-    if [[ $exit_code -ne 0 ]]; then
-        program_exit $exit_code
-    fi
+    # if [[ $exit_code -ne 0 ]]; then
+    #     program_exit $exit_code
+    # fi
```

Comment:
If we don't want that, I think we can simply drop it. What do you think?

Reply:
> If we don't want that, I think we can simply drop it. What do you think?

deleted this part